+neurostim/design.m

```diff
@@ -376,6 +376,11 @@ function show(o,f,str)
             end
         end
         
+        function setConditionOrder(o, condOrder)
+            o.randomization = 'MANUAL';
```

Comment:
@dshimaoka This will work, but it currently has no checks against the current content of the design object. e.g. do all the listed conditions exist? are they consistent with an existing factorial design, for example? The mapping between conditions and factors/levels is tricky and would need to be known by a user to use this meaningfully for designs with more than one factor.